src/components/LoggedCabinet/LoggedCabinet.js

```diff
@@ -9,7 +9,8 @@ import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 
```

Comment:
Please, rename the file and folder to the `logged-cabinet` instead of `LoggedCabinet`

Reply:
Renamed